src/inverseRobot.js

```diff
@@ -24,7 +24,17 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const fixedRobot = {};
+
+  for (const key in robot) {
+    if (fixedRobot[robot[key]]) {
```

Comment:
its better to use `hasOwnProperty` rather than this >` fixedRobot[robot[key]]` CHECKLIST.2